src/test/java/net/minecraftforge/debug/world/MarkDimensionForDeletionTest.java

```diff
@@ -61,7 +61,7 @@
         {
             if (!worldIn.isRemote)
             {
-                if (playerIn.isShiftKeyDown())
+                if (playerIn.isSneaking())
```

Comment:
Why was this additional change made in this PR?

Reply:
because mapping changes made the test not run.